postgres/datadog_checks/postgres/postgres.py

```diff
@@ -716,6 +719,17 @@ def _collect_stats(self, instance_tags):
             with conn.cursor(cursor_factory=CommenterCursor) as cursor:
                 self._query_scope(cursor, archiver_instance_metrics, instance_tags, False)
 
+            if self._config.collect_checksum_metrics and self.version >= V12:
```

Comment:
qq: Would it make sense to log a warn if someone is trying to use this option with postgres < 12? 

If yes, I think only one log message would be enough, no need to spam our logs each time the check runs.

Reply:
I like this idea but I don't think we do this sort of version/config validation currently and want to minimize the blast radius of these changes. Adding it to a ticket for future improvement along with other configuration/version mismatches https://datadoghq.atlassian.net/browse/DBMON-3863

Reply:
Thanks!